openCVLibrary330/build.gradle

```diff
@@ -2,7 +2,6 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 27
-    buildToolsVersion "27.0.3"
```

Comment:
This shouldn't be set to `buildToolsVersion "28.0.3"`?

Reply:
Nope, as mentioned in the PR description the new gradle build tools have their own internal default for `android.buildToolsVersion`. For gradle build tools `3.2.1` that default is `28.0.*`. Not specifying this is fine and reduces the overhead to keeping an app up to date - keep gradle build tools up to date and you'll be up to date with android build tools too